vuu-ui/sample-apps/feature-basket-trading/src/new-basket-panel/NewBasketPanel.tsx

```diff
@@ -76,7 +76,11 @@ export const NewBasketPanel = ({
     <Portal>
       <Popup anchorElement={{ current: document.body }} placement="center">
         <div {...htmlAttributes} className={cx(classBase, className)}>
-          <DialogHeader title="Add New Basket" onClose={onClose} />
+          <DialogHeader
+            title="Add New Basket"
+            onClose={onClose}
+            hideCloseButton={true}
```

Comment:
this could also be just `hideCloseButton` without `={true}` (see other comment on hideCloseButton prop)

Reply:
[2a2dd2f](https://github.com/ScottLogic/finos-vuu/pull/100/commits/2a2dd2ff66595bf9dc891d45a9d49409c7f12bba)